.github/workflows/bigtable-emulator-system-tests.yaml

```diff
@@ -26,7 +26,9 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer update --prefer-dist --no-interaction --no-suggest -d Bigtable/
+          composer config minimum-stability dev -d Bigtable/
+          composer config repositories.local path "../Core" -d Bigtable
+          composer require "google/cloud-core:*" -d Bigtable/
```

Comment:
These additional lines make sure that the tests run the emulator using the current branch, instead of using a tagged version of `google/cloud-core`. This is required for this PR because we made changes in Core that these tests need to pass.